projects/vdk-plugins/vdk-meta-jobs/.plugin-ci.yml

```diff
@@ -1,22 +0,0 @@
-# Copyright 2021-2023 VMware, Inc.
```

Comment:
Let's make changes to vdk-meta-jobs after we have officially released vdk-dags and updated README of vdk-meta-jobs. 

We'd likely deleted the entire directory then.